tempto-runner/src/main/java/com/teradata/tempto/runner/TemptoRunner.java

```diff
@@ -133,7 +133,7 @@ private XmlSuite getXmlSuite()
         XmlClass conventionBasedTestsClass = new XmlClass("com.teradata.tempto.internal.convention.ConventionBasedTestFactory");
         List<XmlClass> classes = newArrayList(conventionBasedTestsClass);
         test.setClasses(classes);
-        test.setParallel(XmlSuite.PARALLEL_METHODS);
+        test.setParallel(XmlSuite.ParallelMode.METHODS);
```

Comment:
We could add reflection here to call the old method if we want to be compatible with both versions. It's unfortunate they didn't keep the old method around for a least one release :(